src/index2.js

```diff
@@ -0,0 +1,4 @@
+const header = document.querySelector('.header')
+console.log(header)
+const dogListHeader = document.querySelector('.dogs-list')
+const dogListBtn = document.querySelector('.dogs-list__button')
```

Comment:
is this needed?

Reply:
I don't know what do you mean exactly Luca?

Reply:
It seems that `index2.js` is not used, am I wrong?

Reply:
yes, actually its not necessary for solution
i just tried to right the code from the beginning